tests/manual/keys-priv/key7

```diff
@@ -1 +1,8 @@
-AGE-SECRET-KEY-169UTU9G4YWRWEW5KTAJEACLHZR2G8TFMW8AUE0S4NRH9R3ZNR66QJW4Y56
\ No newline at end of file
+# This is a comment
```

Comment:
Could you please add this kind of test to the unit tests instead of the manual ones?
Here are the unit tests: https://github.com/slok/agebox/blob/62fc6726188b4a98de4b9f1e3e7a833e0a6bc72b/internal/key/age/age_test.go#L131-L134